spec/migrations/20180130165200_drop_webmks_spec.rb

```diff
@@ -0,0 +1,17 @@
+require_migration
+
+describe DropWebmks do
+  let(:server_stub)   { migration_stub(:MiqServer) }
+  let(:settings_stub) { migration_stub(:SettingsChange) }
+
+  migration_context :up do
+    let(:server) { server_stub.create }
+
+    it 'resets the remote_console_type to default' do
+      settings_stub.create(:resource => server, :key => '/server/remote_console_type', :value => 'MKS')
```

Comment:
Now the tests are failing because you either need to specify `:resource_type` and `:resource_id` or define the relation  in the class defined above.